package.json

```diff
@@ -3,8 +3,9 @@
     "description": "Gruppe Adler coop mission template",
     "version": "2.6.3",
     "dependencies": {
-        "grad-factions": ">0.1.1",
+        "grad-factions": "^3.0.1",
         "grad-fortifications": "^0.9.2",
-        "grad-loadout": ">=4.5.0"
+        "grad-introcam": "^1.0.0",
+        "grad-loadout": "^4.11.5"
```

Comment:
Ich hab hier einfach immer stumpf neueste Version gemacht, weil npm sich nie so verhalten hat, wie ich dachte. 
Ich schätze hier müssten wir das hier machen, damit immer bis vor die nächste major version geupdated wird:

```
        "grad-factions": "^3.x",
        "grad-introcam": "^1.x",
        "grad-loadout": "^4.x"
```

Reply:
@Fusselwurm ?

Reply:
jo, oder noch einfacher

```
        "grad-factions": "3",
        "grad-introcam": "1",
        "grad-loadout": "4"
```

TIL https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

Reply:
hmmm aber mit  `^` geht auch immer also paßt scho, I guess

Reply:
Also kann man es so mergen?

Reply:
schaut so aus.

einfacher test:
* node_modules-Verzeichnis löschen
* im Root-Verzeichnis `npm i` ausführen
* Projekt muß funktionieren ,dh 
    * in Arma laden 
    * richtige Versionen der dependencies geladen haben (also selbe oder neuere solangs keine neue major-versionen sind)
    * package-lock.json kann uU etwas anders aussehen, das wäre egal